plugins/clipboard/plugin.js

```diff
@@ -2345,7 +2345,18 @@
 		if ( editor ) {
 			this.sourceEditor = editor;
 
-			this.setData( 'text/html', editor.getSelectedHtml( 1 ) );
+			var selection = editor.getSelection(),
+				ranges = selection.getRanges(),
+				table = ranges.length && ranges[ 0 ]._getTableElement( { table: 1 } ),
+				element = editor.getSelection().getSelectedElement();
+
+			// When element inside cell is dragged with tableselection plugin on, it is sometimes
+			// nested in additional table. We want to filter only the content of a cell.
+			if ( editor.plugins.tableselection && table && element && element.$.draggable ) {
+				this.setData( 'text/html', element.$.outerHTML );
+			} else {
+				this.setData( 'text/html', editor.getSelectedHtml( 1 ) );
+			}
```

Comment:
This is different issue, it also happens on copy/paste. Please see if it is already reported - if not you can report an issue (and maybe link this piece of code as a possible solution). Then we can proceed separately with it.

Reply:
Reported as #3278. Note that this works fine in FF. It seems to be similar to #972, but that issue is a problem for FF too, so I'm not certain that the cause is the same.

Reply:
PR for the fix - #3290, should be merged before this one. I will remove redundant code during rebase onto latest `major`.

Reply:
After some digging I think #3278 was just some particular case of #972, so I closed it as duplicate. Still, #3290 should be merged before this as it fixes mentioned clipboard issues (also the one mentioned [in another comment](https://github.com/ckeditor/ckeditor4/pull/3244#pullrequestreview-258421144)).